src/test/spec/date-range-picker/date-range-picker.spec.ts

```diff
@@ -512,6 +512,7 @@ describe('DateRangePickerComponent', function(this: ITestContext) {
       const date = '05/04/2020';
       const expectedDate = new Date(date);
       const inputElement = getToElement(this.context.component);
+      inputElement.focus();
```

Comment:
Honestly, this one was a bit confusing as the "from" unit test did not require a focus. I am open to suggestions on if this is an incorrect test.